docs/functional-aspects.md

```diff
@@ -104,10 +104,12 @@ To open the block type `\`. It autopopulates the parameters it is expecting (lik
 
 ## Match
 
-The match expression is used to destructure complex types. It can be used similarly to switch statements in other languages. At the moment, it only supports the Option type (Just and Nothing) and Result type (Ok and Error).
+The match expression is used to destructure complex types. It can be used similarly to switch statements in other languages. It supports the Option type (Just and Nothing) and Result type (Ok and Error).
```

Comment:
```suggestion
The match expression is used to destructure complex types, such as the `Option` type (`Just` and `Nothing`) and the `Result` type (`Ok` and `Error`). It can also be used similarly to switch statements in other languages. 
```